content/graphql/quick-start.md

```diff
@@ -16,7 +16,7 @@ $ npm i @nestjs/graphql @nestjs/apollo graphql apollo-server-express
 # npm i @nestjs/graphql @nestjs/apollo graphql apollo-server-fastify
 
 # For Fastify and Mercurius
-# npm i @nestjs/graphql @nestjs/mercurius graphql mercurius fastify
+# npm i @nestjs/graphql @nestjs/mercurius graphql mercurius 
```

Comment:
Why did you remove fastify?

Reply:
It is not necessary to install it as a dependency, see guide official of Mercurius [here](https://mercurius.dev/#/docs/integrations/nestjs)


<img width="1302" alt="screenshot" src="https://user-images.githubusercontent.com/11542387/216654441-b8a7dce0-0aa7-4bb5-8bee-b0dae5a367f8.png">